htmLawed.php

```diff
@@ -966,7 +966,7 @@ function hl_tag($t)
   if ($t == '>') {
     return '&gt;';
   }
-  if (!preg_match('`^<(/?)([a-zA-Z][^\s>]*)([^>]*?)\s?>$`m', $t, $m)) { // Get tag with element name and attributes
+  if (!preg_match('`^<(/?)([a-zA-Z][^\s>]*)([^>]*?)\s?(/?)>$`m', $t, $m)) { // Get tag with element name and attributes
```

Comment:
Huh, should not this be `\s*`?

Reply:
Actually, in case of unquoted attributes, the space is mandatory, see the note in <https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags>. So, for example, `<hr size=5/>` should get parsed the same as `<hr size="5/">`.

Reply:
Good point, I'll check that :+1:

Reply:
I've added references to the MDN in the commit message and I've updated the pattern (see 2295cf7be7e833214d15720c6b31c4cfed46c597):

![2023-08-12-121846_597x288](https://github.com/fossar/HTMLawed/assets/226063/08b4b4f1-845d-4e9c-ae9a-7d4bb09e2c80)

However, it seems that there's a bug (_unrelated to my PR_) regarding the trailing slash without a space where HTMLawed still consider it as a self-closing tag and add the missing space.